app/controllers/categories_controller.rb

```diff
@@ -0,0 +1,72 @@
+class CategoriesController < ApplicationController
+  before_action :set_category, only: %i[show edit update destroy]
+
+  # GET /categories or /categories.json
+  def index
+    @categories = Category.where(user: current_user)
+  end
+
+  # GET /categories/1 or /categories/1.json
+  # def show
+  #   @transactions1s = @category.transactions1
+  # end
+
+  # GET /categories/new
+  def new
+    @category = Category.new
+  end
+
+  # GET /categories/1/edit
+  def edit; end
+
+  # POST /categories or /categories.json
+  def create
+    @category = Category.new(category_params)
+    respond_to do |format|
+      if @category.save
+        format.html { redirect_to categories_path, notice: 'Category was successfully created.' }
```

Comment:
- [ ] Kindly implement the show route to allow to view a single category, it details and transactions associated to it

Reply:
This issue is well resolved! ✔️